tools/deploy-mainnet.sh

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
```

Comment:
I know I've used the pattern of separate scripts for separate networks before in other repos but since you're taking the time to copy this pattern it is now a good time to consolidate them into one script.  You could either have 3 small stub scripts calling a shared function or one single function called with a parameter set in the makefile.

Reply:
The RPC_URL being set externally will do most of the work for you and so the only thing to configure per network is the chain id param.  The other nice thing you could optionally add while doing this is a default setting of RPC_URL to glif / calibration.glif depending on the network.